using_images/db_images/mysql.adoc

```diff
@@ -189,34 +206,43 @@ stored in the variables and the actual passwords. Whenever a database container
 starts, it resets the passwords to the values stored in the environment
 variables.
 
-You can change these passwords with:
+To change these passwords, update one or both of the desired environment
+variables for the related deployment configuration(s) using the `oc env`
+command:
 
 ----
-$ oc env dc mysql \
+$ oc env dc <dc_name> \
   MYSQL_PASSWORD=<new_password> \
   MYSQL_ROOT_PASSWORD=<new_root_password>
 ----
 
+[IMPORTANT]
+====
+If multiple deployment configurations utilize these environment variables, for
+example in the case of an application created from a template, you must perform
+the above task on each deployment configuration so that the passwords are in
+sync.
+====
+
 This triggers the redeployment of the database server if you have a
 link:../../dev_guide/deployments.html#config-change-trigger[configuration change
 trigger]. Otherwise, you need to manually start a new deployment in order to
 apply the password changes.
 
-Note that you can change one password but not the other by simply omitting one
-of the arguments to `oc env` above.
+To verify that new passwords are in effect, first open a remote shell to the
+running MySQL pod:
+
+----
+$ oc rsh <pod>
+----
 
-You can verify that the new password is in effect with:
+From the bash shell, verify the database user's new password:
 
-====
 ----
-$ oc get pods
-...
-$ oc exec <db_podname> -- bash -c 'mysql -u $MYSQL_USER -pNewPassword -h mysql $MYSQL_DATABASE -te "SELECT * FROM (SELECT database()) db CROSS JOIN (SELECT user()) u"'
+bash-4.2$ mysql -u $MYSQL_USER -p<new_password> -h <db_pod_name> $MYSQL_DATABASE -te "SELECT * FROM (SELECT database()) db CROSS JOIN (SELECT user()) u"
```

Comment:
@rhcarvalho Note that here I updated `-h mysql` to be `-h <db_pod_name>` instead, because when testing this step as is it kept saying "Unknown MySQL server host 'mysql'". Seemed like `-h` should be the hostname of the container (since leaving it off defaults to "localhost"), and `<db_pod_name>` seemed like the easiest way to signal this, since they should be the same. Can you confirm?